modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -2,17 +2,17 @@
 
-== Broker-wide throughput limits
+== Throughput throttling enforcement
 
-The network bandwidth and disk utilization of brokers may be overloaded by clients that produce or consume throughput without limits. To prevent resource overloading caused by unconstrained throughput and to configure back pressure, Redpanda provides runtime-configurable properties that limit and balance throughput of Kafka API traffic.
+Throughput limits are enforced by applying backpressure to clients. When a connection is in breach of the throughput limit, the throttler advises the client about the delay (throttle time) that would bring the rate back to the allowed level. Redpanda starts by adding a `throttle_time_ms` field to responses. If that isn't honoured, delays are inserted on the connection's next read operation. The throttling delay may not exceed the limit set by xref:reference:tunable-properties.adoc#max_kafka_throttle_delay_ms[`max_kafka_throttle_delay_ms`].
 
-To manage the volume of traffic going through a broker, Redpanda implements throughput quotas on the ingress and egress sides of every broker. The throughput quota accounts for all Kafka API traffic going in or out of a broker, with the value of quota representing the allowed rate of data passing through in one direction. When a connection is in breach of the quota, the throttler advises the client about the delay (throttle time) that would bring the rate back to the allowed level, and it implements that delay before handling Kafka API requests. To control the quotas, Redpanda provides configurable rate limits for total ingress and egress traffic through a broker.
+== Broker-wide throughput limits
 
-With Redpanda's xref:get-started:architecture.adoc#thread-per-core-model[thread-per-core model], the Kafka API traffic to and from a client connection is processed by a single core (shard). In order to manage throughput quotas efficiently, broker quotas are distributed between shards, and each per-shard quota is in turn shared by all connections served by the shard. Splitting broker quota optimally between shards is done behind the scenes by the quota balancer component.
+Broker-wide throughput quotas account for all Kafka API traffic going in to or out of a broker, with the value of the quota representing the allowed rate of data passing through in one direction. To control the quotas, Redpanda provides configurable rate limits for total ingress and egress traffic. Redpanda also provides configurable properties which allow excluding clients from throughput throttling and give control over which Kafka request types are subject to throttling limits.
```

Comment:
I trimmed it down further, I've just been trying to avoid the terms ingress and egress here. I know it's wordier but I do think it reads cleaner for the audience:

Broker-wide throughput limits account for all Kafka API traffic going in to or out of the broker. The limit values represent the allowed rate of data in bytes per second passing through in each direction. Redpanda also provides administrators the power to exclude clients from throughput throttling and to fine-tune which Kafka request types are subject to throttling limits.